src/components/Navbar.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import {
+    Box,
+    Flex,
+    HStack,
+    Link,
+    Image,
+    useColorModeValue,
+} from "@chakra-ui/react";
+import { SignInButton } from "@components/SignInButton";
+import { LanguageModal } from "@components/LanguageModal";
+
+type NavbarProps = {
+    session?: unknown;
+};
+
+const Links = [
+    { name: "Browse Programs", url: "#" },
+    { name: "My Classes", url: "#" },
+];
+
+const logoSrc = "/images/logo.png";
```

Comment:
Can you leave a comment so we can go back to it in case we want to convert to svg?